docs/documentation/make-first-prototype/branching.md

```diff
@@ -0,0 +1,78 @@
+# Branching (showing pages depending on user input)
+
+Our first question asks the user how many balls they can juggle. We’re going to send them to an ‘ineligible’ page if they can only juggle 2 or less.
+
+To do this, we’re going to need an ‘ineligible.html’ page, and some logic to decide when to send them there.
+
+This kind of logic goes in a file called `app/routes.js`. Routes tell the kit what to do when the user goes to specific pages, and it’s written in JavaScript.
```

Comment:
Is it the logic or the route that's written in JS?

Reply:
The whole file - up to now the tutorial has been all html and nunjucks. Do you think there's a clearer way of saying this?

Reply:
Maybe we could rearrange these 2 sentences. To something like this, if it conveys the right meaning:

> This kind of logic goes in a file called `app/routes.js`, which is written in JavaScript. Routes tell the kit what to do when the user goes to specific pages.